openassessment/fileupload/backends/filesystem.py

```diff
@@ -0,0 +1,108 @@
+
+from .base import BaseBackend
+from .. import exceptions
+
+from django.conf import settings
+import django.core.cache
+from django.core.urlresolvers import reverse
+
+
+class Backend(BaseBackend):
```

Comment:
OK. My personal preference is for the class to contain the differentiating name - but this is fine.